test.js

```diff
@@ -19,6 +19,29 @@ tape('basic readstream', function (t) {
   })
 })
 
```

Comment:
Can you remove the only

Reply:
Thank you for checking it! Pushed new squashed commit.